tests/data/policy/agent_policy.rego

```diff
@@ -100,7 +88,7 @@ extensions_validated[name] := extension if {
 	extension := object.union(output, {"signingValidated": false})
```

Comment:
We can change this to "signingPolicyValidated". It's trying to convey that the extension signature is validated according to the signing policy rules.

Reply:
Does that mean the signature of the input extension is validated before evaluating whether it needs to be validated? Can’t we skip validation if not required by the rules?

Reply:
now this tag here really means whether the signing policy has been enabled in the rule and whether the extension has met this policy (signed). According to our discussion, if it is signed, and policy didn't request that, we still validate the signature, but will not abort installing and running the extension. Will just log the error. Otherwise, exception will be thrown.

Reply:
I think we should validate all signatures and collect telemetry, but once the feature is stable we should probably only validate when required by policy.

If this tag is indicating that the extension meets the requirement that it is signed (but not necessarily that the signature is validated) then I think we should change the name to indicate that

Reply:
yes, Manu and I suggest this tag is changed to "signingPolicyValidated"

Reply:
Would 'signatureValidationRequired' make sense?

Reply:
it could be rule doesn't request signing, but the extension has been signed. or the rule doesn't request signing, and the extension is not signed, in both cases signingPolicyValidated will be true. if changed to signatureValidationRequired, it is doesn't cover the latter case.